lib/process-services-cloud/src/lib/form/components/form-cloud.component.scss

```diff
@@ -0,0 +1,98 @@
+/* stylelint-disable scss/at-extend-no-missing-placeholder */
+.adf-full-screen {
+    width: 100%;
+    height: 100%;
+    background-color: var(--adf-theme-background-card-color);
+}
+
+.adf-cloud-form {
+    &-container {
+        .adf-cloud-form-content {
+            @extend .adf-full-screen;
+
+            flex: 1;
+            flex-direction: column;
+            display: flex;
+        }
+    }
+
+    &-overlay-container {
+        .adf-cloud-form-content {
+            position: fixed;
+            top: 0;
+            left: 0;
+            z-index: 100000;
+        }
+    }
+
+    &-inline-container {
+        @extend .adf-full-screen;
+    }
+
+    &-toolbar {
+        .mat-toolbar {
+            background-color: var(--adf-theme-background-card-color-087);
+        }
+    }
+
+    &-fullscreen-button {
+        position: absolute;
+        right: 70px;
+        top: 30px;
+    }
+
+    &__display-name {
```

Comment:
Are we sure, we are not causing any breaking changes with these?

Reply:
We are using a brand new selector prefix (`.adf-cloud-form`), so I don't think we can cause issues